crates/subspace-runtime/src/lib.rs

```diff
@@ -121,7 +121,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: Cow::Borrowed("subspace"),
     impl_name: Cow::Borrowed("subspace"),
     authoring_version: 0,
-    spec_version: 2,
+    // TODO: before deploying the next runtime version to mainnet or taurus, bump this version
+    spec_version: 11,
```

Comment:
even this one, let not bump it. 
when wwe create a taurus upgrade branch, we can bump it then. 
TODO might be noise here

Reply:
We did bring this up earlier but I think we will continue using branches for Taurus as we did not completely align on this yet. Until then lets follow the current flow of having seperate taurus branch
We usually do spec bump in a seperate PR